service-front/module/Application/src/Validators/LpaValidator.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Validators;
+
+use Laminas\Validator\AbstractValidator;
+
+class LpaValidator extends AbstractValidator
```

Comment:
Could potentially be renamed to `LpaUidValidator` for clarity (since it's not validating the whole LPA)